app/src/main/java/com/work/restaurant/view/calendar/CalendarFragment.kt

```diff
@@ -0,0 +1,121 @@
+package com.work.restaurant.view.calendar
+
+import android.app.DatePickerDialog
+import android.os.Build
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.DatePicker
+import android.widget.TextView
+import android.widget.Toast
+import com.work.restaurant.R
+import com.work.restaurant.util.App
+import com.work.restaurant.view.base.BaseFragment
+import kotlinx.android.synthetic.main.calendar_main.*
+import java.text.SimpleDateFormat
+import java.util.*
+
+class CalendarFragment : BaseFragment(R.layout.calendar_main),
+    View.OnClickListener {
+
+
+    override fun onClick(v: View?) {
+
+        when (v?.id) {
+
+        }
+    }
+
+    var button_date: Button? = null
+    var textview_date: TextView? = null
+    var cal = Calendar.getInstance()
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.calendar_main, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+
+        textview_date = this.text_view_date_1
+        button_date = this.button_date_1
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            date_Picker.setOnDateChangedListener { view, year, monthOfYear, dayOfMonth ->
+                Toast.makeText(
+                    this.context,
+                    "$year ${monthOfYear + 1} $dayOfMonth",
+                    Toast.LENGTH_SHORT
+                )
+                    .show()
+            }
+        }
+
+
+
+
+
+
+        calender?.setOnDateChangeListener { view, year, month, dayOfMonth ->
```

Comment:
```suggestion
        calender.setOnDateChangeListener { view, year, month, dayOfMonth ->
```
null check 할 필요 없어보여요

Reply:
e9c0f82 수정하였습니다.